src/modules/destination.js

```diff
@@ -67,4 +67,21 @@ const addHotels = async (budget, trip) => {
   }
 };
 
-export { getCoords, initializeTrip, addActivityType, addHotels };
\ No newline at end of file
+const addRestaurants = async (preference, budget, trip) => {
+  try {
+    const response = await axios({
+      method: "POST",
+      url: "api/v1/hotels",
+      params: {
+        trip: trip,
+        preference: preference,
+        budget: budget
+      }
```

Comment:
Make the request to `api/v1/activity_types`
In order to avoid manipulating the params as much as possible before forwarding the request to Google Places API I suggest the following params:
```
params: {
        trip: trip,
        keyword: preference,
        max_price: budget,
        activity_type: "restaurant"
      }
```

Reply:
or eh, call them what you want it doesn't matted, but include the activity_type as restaurant

Reply:
no problem